libraries/CMakeLists.txt

```diff
@@ -11,3 +11,12 @@ endif()
 
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/pbrlib/genosl/pbrlib_genosl_impl.${PBRLIB_SUFFIX}"
         DESTINATION "${MATERIALX_INSTALL_STDLIB_PATH}/pbrlib/genosl/" RENAME pbrlib_genosl_impl.mtlx)
+
+if (MATERIALX_INCLUDE_PYTHON_STDLIB)
```

Comment:
All the option does is copy over the libraries into `python/MaterialX`
which gets picked up by `setup.py`